python/tests/table_html/test_to_html_dataarray.py

```diff
@@ -174,15 +174,18 @@ def test_bin_edge_and_sparse(dims, lengths):
     in_unit = sc.units.m
     in_dtype = sc.dtype.event_list_float32
 
-    data = sc.Variable(dims, lengths, unit=in_unit, dtype=in_dtype)
+    data = sc.Variable(dims=dims, shape=lengths, unit=in_unit, dtype=in_dtype)
 
     # attribute data without the sparse dimension
-    non_sparse_data = sc.Variable(dims, lengths, unit=in_unit, dtype=in_dtype)
+    non_sparse_data = sc.Variable(dims=dims,
+                                  shape=lengths,
+                                  unit=in_unit,
+                                  dtype=in_dtype)
 
     # makes the first dimension be bin-edges
     lengths[0] += 1
-    non_sparse_bin_edges = sc.Variable(dims,
-                                       lengths,
```

Comment:
Are you implying that this would now call the quaterion constructor?

Reply:
No it's not calling the quaterinon constructor, it was failing with an error message that was talking about eigen, but maybe I should change that.
Basically when you call without keywords and you give a `list` as first argument as well as a `dtype`, it believes that the list is for the `values`, and tries to see if `dtype` is either `vector_3_float` or `quaternion_float`. If it is not, it fails with the message saying `"Wrong overload for making 0D Eigen variable."` but actually that should probably be changes to something like `"Cannot create Variable from value list with dtype {dtype}"?

Reply:
Do we want that constructor
```
non_sparse_bin_edges = sc.Variable(dims,
                                       lengths,
```
to work like it did before, or is it ok to have to specify the argument names?

Reply:
I'm ok with how you did it.